spec/features/items/index_spec.rb

```diff
@@ -39,8 +44,6 @@
         expect(page).to have_link(@hippo.name)
         expect(page).to have_content(@hippo.description)
         expect(page).to have_content("Price: #{number_to_currency(@hippo.price)}")
-        expect(page).to have_content("Active")
-        expect(page).to have_content("Inventory: #{@hippo.inventory}")
```

Comment:
Between lines 42 and 43, please add the following line: 
expect(page).to have_content("Inventory: #{@hippo.inventory}")